lib/decoder.js

```diff
@@ -747,6 +747,13 @@ var JpegImage = (function jpegImage() {
               successiveApproximation >> 4, successiveApproximation & 15);
             offset += processed;
             break;
+
+            case 0xFFFF: // Fill bytes
```

Comment:
nit: feel free to change
Example bad file attached - this was one from a CCTV system that tripped me up.
![mpt dgos_camera3_mpt dgos_20170813152826_3955415](https://user-images.githubusercontent.com/1282156/29426922-052e6b2e-8380-11e7-9c65-81f330bf6dc8.jpg)